tools/preprocess_reds_dataset.py

```diff
@@ -160,14 +162,24 @@ def unzip(zip_path):
     """
     zip_files = mmcv.scandir(zip_path, suffix='zip', recursive=False)
     import zipfile
+    import shutil
     unzip_folders = []
     for zip_file in zip_files:
         zip_file = osp.join(zip_path, zip_file)
         unzip_folder = zip_file.replace('.zip', '').split('_part')[0]
-        unzip_folder = osp.join(zip_path, unzip_folder)
         print(f'Unzip {zip_file} to {unzip_folder}')
         with zipfile.ZipFile(zip_file, 'r') as zip_ref:
             zip_ref.extractall(unzip_folder)
+        data_name = osp.basename(unzip_folder)
+        data_type = data_name.split('_')[0]
+        # if data path like `train_sharp/train/train_sharp/*`
+        # begin reorganizing to `train_sharp/*`
+        if osp.isdir(osp.join(unzip_folder, data_type, data_name)):
+            data_folder = osp.join(unzip_folder, data_type, data_name)
+            for i in os.listdir(data_folder):
+                shutil.move(osp.join(data_folder, i), unzip_folder)
+        shutil.rmtree(osp.join(unzip_folder, data_type))
```

Comment:
please move this line under `if osp.isdir(osp.join(unzip_folder, data_type, data_name)):`

Reply:
Since I have not merged this PR, could you just modify it so that I can merge it in this PR?

Reply:
done.